### PR TITLE
Added UI btn | next, prev, workweek, edit , delete

### PR DESCRIPTION
**Pull request by Mansimran-Singh**

### CODE REVIEW

File: functions/views/pages/calendar.ejs [hunk 1]
```diff
@@ -1,69 +1,191 @@
...
+    <%- include('../partials/head') -%>
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://uicdn.toast.com/tui-calendar/latest/tui-calendar.css"
+    />
+
+    <!-- If you use the default popups, use this. -->
+    <link
```
Comment (mbeclawski) [inline functions/views/pages/calendar.ejs]: why are you including this? we already had it (locally)

File: functions/views/pages/calendar.ejs [hunk 2]
```diff
@@ -1,69 +1,191 @@
...
+      rel="stylesheet"
+      type="text/css"
+      href="https://uicdn.toast.com/tui.date-picker/latest/tui-date-picker.css"
+    />
+    <link
```
Comment (mbeclawski) [inline functions/views/pages/calendar.ejs]: why are you including this? we already had it

File: functions/views/pages/calendar.ejs (hunk 1 above)
Comment (Mansimran-Singh) [reply functions/views/pages/calendar.ejs]: The commit I pulled from had bower something links and wasn't loading calendar locally for me so adding these loaded it. Feel free to add only stuff needed

Comment (Mansimran-Singh) [reply functions/views/pages/calendar.ejs:15]: Oh i see what you mean, i probably accidentally added that link tag twice, please get rid of one

**Pull request merged**